### PR TITLE
Laravel compatibility

### PR DESCRIPTION
**Pull request by martijngastkemper**
To prevent conflicts between the Laravel helpers some Garp functions will be removed. Almost all functions have a Garp Functional equivalent. Some, `view()` for example, haven't, but can't be available as global function.

### CODE REVIEW

**Comment by martijngastkemper** [comment]
Review remarks:
- Some functions arguments in Garp Functional have a different order. For example `prop` and `pick`.
- To make the linter happy I had to update the code style. It's all in separate commits.
- Test `Garp_Content_Api_Rest::post` very carefully.

**Comment by martijngastkemper** [comment]
Maybe out of scope for this PR, but `array_column` and `gzdecode` polyfills could be removed, because there are available in the minimal required PHP version.

File: library/Garp/Content/Api/Rest.php [hunk 1]
```diff
@@ -55,20 +58,12 @@ public function post(array $params, array $postData) {
...
-            ),
-            'result',
-            array()
-        );
+        $response = $this->get([
+            'datatype' => $params['datatype'],
+            'id' => $primaryKey,
+        ]);
+
+        $record = f\either(f\prop_in(['result', 'data']), [])($response);
```
Comment (harmenjanssen) [inline library/Garp/Content/Api/Rest.php]: These are the wrong way around. 🙂 
Verified in Subsidieportaal.

Comment (harmenjanssen) [reply library/Garp/Content/Api/Rest.php]: Should be `f\prop_in(['data', 'result'])`.

**Comment by harmenjanssen** [comment]
> Maybe out of scope for this PR, but `array_column` and `gzdecode` polyfills could be removed, because there are available in the minimal required PHP version.

Yeah definitely, they could've been removed a while ago. 🙂

**Comment by martijngastkemper** [comment]
I've removed polyfills `array_column` and `gzdecode`.

**Pull request merged**